### DISCUSSION TITLE
Can't get the Realtek RTL8029AS network driver to work (Windows NT 3.51)

### DISCUSSION THREAD

**Discussion by unbreakify** [86Box/86Box#3773] [Q&A]
I've been trying to install this damn network driver for ages, i do everything right but it always bothers me to "insert the Realtek RTL8029 OEM Network Driver disk". I don't have a damn "network driver disk" - and i never did. **The files for what it needs are there, OEMSETUP.INF & RTL8029.SYS** , but it doesnt read it.
![image](https://github.com/86Box/86Box/assets/131456819/1d01e714-e348-49ca-baa7-01eac90102d8)
![image](https://github.com/86Box/86Box/assets/131456819/ea4f64f9-adb7-4b24-8508-fe934fd7b458)
![image](https://github.com/86Box/86Box/assets/131456819/736a3264-0419-489e-8a80-33f6ce06a636)
![image](https://github.com/86Box/86Box/assets/131456819/c70870bf-0f84-4806-9140-8d82811bc91d)

**Comment by TechGuyC** [comment]
Have you tried checking the Floppy disk setting in the bios? Have you tried using another motherboard of the same socket?

**Reply by unbreakify** [reply]
The bios settings don't _seem_ to be the problem, however i could be wrong - the motherboard i'm using right is a [SiS] Soyo 4SAW2 using an i486 (Socket 3)

**Reply by TC1995** [reply]
You that empty/dummy disk1 file it expects on the root of the floppy drive or something (see its oemsetup.inf).
